utils/subxt/codegen/README.md

```diff
@@ -5,7 +5,25 @@ Use this as a library or binary to generate subxt types.
 ### Using the binary
 
 ```bash
- cargo build --release -p codegen --bin codegen 
  # ensure that you have the necessary nodes running in the background, reachable on ports
- ./target/release/codegen --path ./utils/subxt/generated/src
-```
\ No newline at end of file
+ cargo run --release -p codegen --bin codegen -- --path ./utils/subxt/generated/src/default
+```
+
+### Adding new chain
+1. Create new folder (e.g. `new_chain`) in `utils/subxt/generated/src` with `mod.rs`:
+```rust
+pub mod parachain;
+pub mod relaychain;
+```
+2. Generate types for the chain:
+```bash
+ cargo run --release -p codegen --bin codegen -- --path ./utils/subxt/generated/src/new_chain
+```
+3. Create a corresponding config in `hyperspace/core/substrate/new_chain.rs` (see `default.rs` for example)
+4. Add a variant to all the `Any*` enums in `hyperspace/core/chain.rs`. For example,
+```rust
+pub enum AnyChain {
+	Parachain(ParachainClient<DefaultConfig>),
+	NewChain(ParachainClient<NewChainConfig>), // <-- newly added variant
```

Comment:
Yes, that's the only way to add a new chain now. But it's just 1 module of code

Reply:
It's not one line. Later, it will be nearly 1-line, but it requires adding a macro, which is a separate work. Personally, I don't see any issues spending 2-3 minutes on copy-pasting code to add a new chain. 
I've removed all the `_ => unreachable!()` branches.
You can run 2 instances of hyperspace to relay between 4 chains.